pkg/minikube/sshutil/sshutil.go

```diff
@@ -63,6 +63,29 @@ func NewSSHClient(d drivers.Driver) (*ssh.Client, error) {
 	return client, nil
 }
 
+func NewRawSSHClient(ip, sshKeyPath, username string) (*ssh.Client, error) {
```

Comment:
why do we need this?
It is an SSH client without the driver involved, right?

Reply:
> It is an SSH client without the driver involved, right?

yes, that's right.